sdk/java/src/main/java/com/pulumi/aws/lightsail/inputs/BucketState.java

```diff
@@ -78,14 +78,14 @@ public Optional<Output<String>> createdAt() {
     }
 
     /**
-     * Force Delete non-empty buckets using `TODO destroy`. AWS by default will not delete an s3 bucket which is not empty, to prevent losing bucket data and affecting other resources in lightsail. If `force_delete` is set to `true` the bucket will be deleted even when not empty.
+     * Force Delete non-empty buckets using `pulumi down`. AWS by default will not delete an s3 bucket which is not empty, to prevent losing bucket data and affecting other resources in lightsail. If `force_delete` is set to `true` the bucket will be deleted even when not empty.
```

Comment:
TIL pulumi down.

Reply:
That was a mistake. It should be `pulumi destroy`, but `pulumi down` does work.